kie-pmml-trusty/kie-pmml-models/kie-pmml-models-scorecard/kie-pmml-models-scorecard-tests/src/test/java/org/kie/pmml/models/scorecard/tests/SimpleScorecardCategoricalTest.java

```diff
@@ -81,4 +86,25 @@ public void testSimpleScorecardCategorical() {
         Assertions.assertThat(pmml4Result.getResultVariables().get(REASON_CODE1_FIELD)).isEqualTo(reasonCode1);
         Assertions.assertThat(pmml4Result.getResultVariables().get(REASON_CODE2_FIELD)).isEqualTo(reasonCode2);
     }
+
+    @Test
+    public void testSimpleScorecardCategoricalVerifyNoException() {
+        final List<Map<String, Object>> samples = getSamples();
```

Comment:
```suggestion
        getSamples().stream().map(sample -> evaluate(pmmlRuntime, sample, MODEL_NAME)).forEach(Assert::assertNotNull);
```
the rest of the method disappears

Reply:
the same for SimpleScorecardCategoricalTest.java